README.md

```diff
@@ -99,31 +65,30 @@ To use the Material theme, import the correspondent file from the `theme/materia
 
 - The component with the Lumo theme:
 
-  `theme/lumo/vaadin-select.html`
+  `theme/lumo/vaadin-select.js`
 
 - The component with the Material theme:
 
-- `theme/material/vaadin-select.html`
+- `theme/material/vaadin-select.js`
 
-- Alias for `theme/lumo/vaadin-select.html`:
+- Alias for `theme/lumo/vaadin-select.js`:
 
-- `vaadin-select.html`
+- `vaadin-select.js`
 
 
 ## Running demos and tests in a browser
 
 1. Fork the `vaadin-select` repository and clone it locally.
 
-1. Make sure you have [npm](https://www.npmjs.com/) and [Bower](https://bower.io) installed.
+1. Make sure you have [npm](https://www.npmjs.com/) installed.
 
-1. When in the `vaadin-select` directory, run `npm install` and then `bower install` to install dependencies.
+1. When in the `vaadin-select` directory, run `npm install` to install dependencies.
```

Comment:
For some reason `npm i` doesn't finish successfully

Reply:
Which node version do you have? Is there any error in the console? Everything works for me with Node 12.18.